src/app/shared/data-access/event.service.ts

```diff
@@ -65,10 +137,79 @@ export class EventService {
       ),
     );
 
-  readonly events = toSignal(this.events$);
-  readonly eventForThisRound = toSignal(this.eventForThisRound$);
   readonly eventForNextRound = toSignal(this.eventForNextRound$);
 
+  readonly events = toSignal(this.events$);
+
+  readonly eventTeams$: Observable<EventTeamModel[] | null> = this.events$.pipe(
+    map((events) => events?.map((ev) => ev.id)),
```

Comment:
I believe using optional chaining `?.` always returns `undefined`, event if the value is null. If that's true, then the `whenNotNull` function wouldn't work how we expect, since the returned value would be `undefined` instead of `null` We could fix it by adding `?? null`:

```typescript
map((events) => events?.map((ev) => ev.id) ?? null),
```

Reply:
Makes sense.  Added `?? null`.